valter-junior/projetoIP
# (Feature): ClassVida

**What I did**

- I created the class Vida
  - the class vida will be used in main

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout main`
   2. `git pull`
   3. `git checkout feature/cassvida`
   4. `git pull origin feature/cassvida`
   
3. Test the things that you think are worth testing, even the ones that are not in this description
   - test if the is working how expected; 
   - test if it's working on the main;
